src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid implements Figure {
```

Comment:
The class IsoscelesTrapezoid should not begin with an empty line. Please remove the redundant empty line at the start of the class definition.